Gemfile

```diff
@@ -21,12 +19,19 @@ gem 'jbuilder', '~> 2.5'
 # gem 'bcrypt', '~> 3.1.7'
 
 gem 'config'
+# Use Capistrano for deployment
+# gem 'capistrano-rails', group: :development
+
+# useful for debugging, even in prod
+gem 'pry-byebug' # Adds step-by-step debugging and stack navigation capabilities to pry using byebug
+gem 'pry-rails' # use pry as the rails console shell instead of IRB
```

Comment:
i've definitely used rails console a number of times on prod to investigate argo issues.  not a thing that i'd expect to do often, but something i'd expect to come up at least some.  e.g., you want to run load up some objects via ActiveRecord to because some error cropped up when a thing you expected to be there wasn't.  and in that case, the syntax highlighting and formatting and inter-session history and other niceties of pry are great.